### PR TITLE
clean unuse print

### PR DESCRIPTION
**Pull request by rokkiter**
**Please provide a description of this PR:**

clean code: this println seem unnecessary

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Review by hanxiaop — CHANGES_REQUESTED** [review]
It's necessary, it separates different summaries.

**Comment by rokkiter** [comment]
yes you are right, it can separates different summaries
image
but it contain Stderr 
should we use fmt.println() to replace it

**Comment by hanxiaop** [comment]
> yes you are right, it can separates different summaries image but it contain Stderr should we use fmt.println() to replace it

It maynot be right I think. We should use `OutOrStdout `

**Comment by rokkiter** [comment]
good idea, i have change to OutOrStdout

File: istioctl/cmd/injector-list.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func injectorListCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			cmd.Println()
+			fmt.Fprint(cmd.OutOrStdout(), fmt.Sprintln())
```
Comment (hanxiaop) [inline istioctl/cmd/injector-list.go:110]: Why not just using `fmt.Fprintln`?

Comment (hanxiaop) [reply istioctl/cmd/injector-list.go:110]:
```suggestion
			fmt.Fprintln(cmd.OutOrStdout())
```

Comment (rokkiter) [reply istioctl/cmd/injector-list.go]: I'm using OutOrStdout following the previous format, but it really doesn't seem necessary

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**